tests/fixtures/response.json

```diff
@@ -0,0 +1 @@
+{"id":"0","run_id":"0","queued":110,"skipped":0,"errors":[],"run_url":"https://buildkite.com/organizations/allurion/analytics/suites/collector-test/runs/0"}
```

Comment:
While not against it, I don't think it is necessary to hardcode your actual organization URL, just put an `example.com` one